napari/components/_tests/test_viewer_model.py

```diff
@@ -125,7 +125,7 @@ def test_single_point_dims():
     shape = (1, 3)
     data = np.zeros(shape)
     viewer.add_points(data)
-    assert all(r == (0.0, 1.0, 1.0) for r in viewer.dims.range)
+    assert all(r == (-5.0, 6.0, 1.0) for r in viewer.dims.range)
```

Comment:
I think it is a little less clear what `dims.range` should be for points than for image-based layers (see for example discussion here: https://github.com/napari/napari/pull/3444#pullrequestreview-769450434.

As a concrete example, if I check out this PR branch and run the following script that generates a single 3D point at the origin, nothing is visible when it first opens, but if I scroll the slider to 5th position to see the point.

```Python
import napari
import numpy as np

with napari.gui_qt():
    viewer = Viewer()

    one_pixel = np.ones((1, 1, 1))
    points = np.zeros(3)
```

I suppose, if the change proposed in #3559 (set initial point to the centroid) was also made it would make the point visible upon first opening the viewer, though.